front/app/components/PostShowComponents/Comments/messages.ts

```diff
@@ -9,6 +9,14 @@ export default defineMessages({
     id: 'app.components.Comments.cancel',
     defaultMessage: 'Cancel',
   },
+  edit: {
+    id: 'app.components.Comments.editComment',
+    defaultMessage: 'Edit',
+  },
+  delete: {
+    id: 'app.components.Comments.deleteComment',
+    defaultMessage: 'Delete',
+  },
```

Comment:
How come we have new messages for this?

Reply:
It was a mistake - removed now! Thanks for spotting :)